.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -1,4 +1,5 @@
 name: Feature
+type: 'Bug'
```

Comment:
Is this right? This is the feature request template. I would think we'd want to use `Feature`. I see we have these types currently enabled:

<img width="1171" alt="Screenshot 2025-01-28 at 11 42 44 AM" src="https://github.com/user-attachments/assets/987cec4e-452e-4bab-adf9-5f46cac03731" />